inference-engine/tests/functional/plugin/shared/models/max_pool_qdq.prototxt

```diff
@@ -0,0 +1,126 @@
+ir_version: 6
```

Comment:
I don't like the approach to store models under git. And what is the license of the model? Is there the way not to store this model as a file under git?

Reply:
We use this approach for a long time, for example:
https://github.com/openvinotoolkit/openvino/tree/master/inference-engine/tests/functional/inference_engine/onnx_reader/models
https://github.com/openvinotoolkit/openvino/tree/master/ngraph/test/models